src/include/uservarmanager.h

```diff
@@ -40,6 +55,19 @@ class DLLIMPORT UserVariableManager : public Mgr<UserVariableManager>
 
         UserVariableManager& operator=(const UserVariableManager&) = delete;
         UserVariableManager(const UserVariableManager&) = delete;
+
+        static const wxString cBase;
+        static const wxString cDir;
+        static const wxChar   cSlash;
+        static const wxString cSlashBase;
+        static const wxString cInclude;
+        static const wxString cLib;
+        static const wxString cObj;
+        static const wxString cBin;
+        static const wxString cCflags;
+        static const wxString cLflags;
+        static const wxString cSets;
+        static const wxArrayString builtinMembers;
```

Comment:
Why are these made public? I don't see them using them. If you use them for something add a block doxygen comment stating the reason.